public/index.html

```diff
@@ -0,0 +1,179 @@
+              badge on your products and documentation pages.
+            </p>
+          </div>
+          <div class="offset-md-1 col-md-4 pb-2">
+            <img
+              src="images/check-decagram-outline.svg"
+              height="35"
+              class="mb-4 d-inline-block"
+            />
+            <h4>Verified</h4>
+            <p>
+              Shown on all integration pages on the Home&nbsp;Assistant Website
+            </p>
+          </div>
+          <div class="offset-md-1 col-md-4 pb-2">
+            <img
+              src="images/magnify.svg"
+              height="35"
+              class="mb-4 d-inline-block"
+            />
+            <h4>Searchability</h4>
+            <p>
+              Over 500,000 Home&nbsp;Assistant users that search the website
+              daily for new integrations.
+            </p>
+          </div>
+          <div class="offset-md-1 col-md-4 pb-2">
+            <img
+              src="images/thumb-up-outline.svg"
+              height="35"
+              class="mb-4 d-inline-block"
+            />
+            <h4>Social Media</h4>
+            <p>
+              Using Home&nbsp;Assistant's Blog and Social Media accounts to
+              promote the partnership.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section alt">
+      <div class="container">
+        <div class="row">
+          <div
+            class="partner-images d-md-flex justify-content-center align-items-end py-3"
+          >
+            <img src="images/esphome-logo-text.png" width="200" />
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section">
+      <div class="container">
+        <div class="row">
+          <div
+            class="col-md-5 text-center d-md-flex justify-content-center align-items-center"
+          >
+            <p class="display-6">
+              What does it take to be a Works with Home&nbsp;Assistant Partner?
+            </p>
+          </div>
+          <div class="offset-md-1 col-md-6 lead">
+            <ul>
+              <li>Integration must be configurable via the user interface</li>
+              <li>If cloud based, account linking must be supported</li>
+              <li>Devices tested and approved by Nabu Casa</li>
+              <li>Integration Quality Scale requirement of Gold or better</li>
+              <li>Support contact and Integration Maintainer must be listed</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section">
+      <div class="container">
+        <div class="row">
+          <div class="offset-md-2 col-md-8">
+            <p class="display-4 mb-4">Responsibilities</p>
+            <p class="faq-q">Response to support inquiries</p>
+            <p>
+              As issues may arise with your integration, Nabu Casa will reach
```

Comment:
Do we reach out? Isn't it better to let the bot handle the codeowner ship notifications? Or are we going to email them about every GitHub issue reported in our issue tracker?

Reply:
Not sure what you mean. I am saying.

The issue may not be with the code itself.

The issue could be with the product.

Leviton doesn't code own so we would need to contact.

This isn't a contract. They will need to meet with me to understand the full requirements. We are generic here

Reply:
As issues may arise with your integration, your responsibility is
              to respond to these support inquiries in a quick and timely
              manner. The support contact for Z-Wave, Zigbee, or Matter partners
              will be expected to support any inquiries made by the Nabu Casa
              team.

Reply:
Thoughts?

I think this maybe be more inline with both paragraphs